README.md

```diff
@@ -57,24 +50,47 @@ With simple commands, this bot allows you to:
 
 [![Demo Video](https://i9.ytimg.com/vi/aCjcT1CHAzU/mq3.jpg?sqp=COSotI0G&rs=AOn4CLD34jFIlq6GRdmTnK6p3F8O2F-Yig)](https://youtu.be/aCjcT1CHAzU)
 
-## What is new in this version?
+## What's new?
+
+<#TODO: short description here>
 
 Checkout the [this documentation](https://github.com/usmanwardag/dollar_bot/blob/main/docs/Updated_version.pdf)
+
 ## Installation
 
-The below instructions can be followed in order to set-up this bot at your end in a span of few minutes! Let's get started:
+#### Pre-requisite: The Telegram Desktop App
 
-1. Clone this repository to your local system.
+Since DollarBot is built on top of Telegram, you'll first need:
+1. Download the Telegram Desktop Application <a href="https://desktop.telegram.org/">here.</a>
+```https://desktop.telegram.org/```
+2. Create a Telegram account or Sign in.
 
-2. Start a terminal session in the directory where the project has been cloned. Run the following command to install the required dependencies:
-```
-  pip install -r requirements.txt
+Open up your terminal and let's get started:
 
+### MacOS / Ubuntu Users
+
+1. Clone this repository to your local system. 
+```
+   git clone https://github.com/aditikilledar/dollar_bot_SE23/
 ```
-3. Download and install the Telegram desktop application for your system from the following site:
- ```
- https://desktop.telegram.org/
- ```
+2. Start a terminal session in the directory where the project has been cloned. Run the following command and follow the instructions on-screen to complete the installation.
```

Comment:
BotFather step should come either before or between step one and two. Slightly confusing when explained after platform specific instructions.

Reply:
Made it clearer. Please review and get back to me.